extensions/default/src/Panels/PanelStudyBrowser.jsx

```diff
@@ -1,6 +1,8 @@
 import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
-import { StudyBrowser, useImageViewer } from '@ohif/ui';
+import { StudyBrowser, useImageViewer, utils } from '@ohif/ui';
```

Comment:
Since we are using getFormattedDate inside the UI package and some extensions may not use ohif/ui, I think this util should not live there. Can we just:
```var todayDate = new Date().toISOString().slice(0,10);``` or maybe move this util to ohif/core/utils?

Reply:
I agree that it shouldn't live in `@ohif/ui`.
`@ohif/core/utils` is better

I'd rather we not use `toISOString()` --> date-fns and moment are more robust, and JavaScript dates are notoriously awful.

Reply:
Thanks guys! this util was moved to `@ohif/core`, still using moment.